### PR TITLE
Create Content on Community Section

### PR DESCRIPTION
**Pull request by bmath10**
# Summary
Four new pages have been added to the community section of the documentation site.

# Testing Guidance

- Run `bundle exec jekyll serve`
- Navigate to `localhost:4000/index_new.html`, then go to the `Community` page.
- Confirm that the content in the community section is accurate

### CODE REVIEW

File: community/test-kits.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+---
+title: Test Kits
+nav_order: 2
+layout: docs
+section: community
+---
+
+# Test Kits
```
Comment (arscan) [inline community/test-kits.md:8]: Hmmm, shoudl this be called 'Test Kit Registry'?

File: community/test-kits.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+Here is the list of Test Kits that the Inferno Team is currently aware of, along with a link to the source code and where it is publicly hosted, if available.
```
Comment (arscan) [inline community/test-kits.md:10]: Could this language be cleaned up a little, feels very informal.  Also, could you add in a link here, which shows inferno test kits that are publicly available on github: https://github.com/inferno-framework/inferno-core/network/dependents

**Comment by arscan** [comment]
I think we should work in here somewhere a list of available public hosts.  That could either go in the 'Test Kit' section (simply because we reference the only known public host).  Perhaps adding a link to Inferno.healthit.gov qa server would be worthwhile, as SOMETIMEs we want people to go there for connectathons, etc... so we would have 2 public hosts i suppose :)

**Comment by arscan** [comment]
We also need to work in 'Additional Tools' or 'Inferno Add-ons' section.  Which basiscally will just be the 'inferno reference server' (which really, we should start calling the 'inferno us core server simulation'), the fhir-validator-wrapper, and the validator-app.

**Review by arscan — APPROVED** [review]
I'll go ahead and approve.  But I think we'll probably want to address my comments at some point, if not in this PR than in another revision.

File: community/test-kits.md (hunk 2 above)
Comment (bmath10) [reply community/test-kits.md:10]: can do! Are there other test kits at that link that should be included in the table?

**Comment by bmath10** [comment]
> We also need to work in 'Additional Tools' or 'Inferno Add-ons' section. Which basiscally will just be the 'inferno reference server' (which really, we should start calling the 'inferno us core server simulation'), the fhir-validator-wrapper, and the validator-app.

I feel like Additional Tools would go under Documentation, rather than Community?

**Review by arscan — APPROVED** [review]
Could you add years to all the tech talk ppts?  Otherwise go ahead and merge!

**Pull request merged**